session-replacement/src/main/java/com/amadeus/session/servlet/CookieSessionTracking.java

```diff
@@ -57,6 +57,10 @@
    * to javascript.
    */
   static final String COOKIE_HTTP_ONLY_PARAMETER = "com.amadeus.session.cookie.httpOnly";
+  /**
+   * Used to specify domain or path on witch will be applied the cookie
+   */
+  static final String COOKIE_PATH_PARAMETER = "path";
```

Comment:
See comment in WebXmlParser. Otherwise prefix `com.amadeus.session`  is mandatory. You might need to adapt cookiePath() method and use context path if it was specified.